apps/consent/app/components/captcha-challenge/index.tsx

```diff
@@ -35,18 +54,21 @@ const CaptchaChallengeComponent: React.FC<{
   );
 
   useEffect(() => {
-    // @ts-ignore
-    window.initGeetest(
-      {
-        gt: id,
-        challenge: challenge,
-        offline: false,
-        new_captcha: true,
-        product: "bind",
-        lang: "en",
-      },
-      captchaHandler
-    );
+    if (process.env.NODE_ENV === "development") {
```

Comment:
Do we need to add this to env.ts? This is handled by Next.js itself.

Reply:
I don't think we are doing to use the `next` binary but rather `node` directly so it may just not be here (in prod at least). @bodymindarts had some comment on it today or yesterday

Reply:
anyhow I think that make sense to declare the env variable we use in a single place